static/app/types/alerts.tsx

```diff
@@ -1,4 +1,5 @@
 import {IssueConfigField} from 'app/types/index';
+import {Config} from 'app/views/organizationIntegrations/sentryAppExternalForm';
```

Comment:
Is config a good name for this type if we're going to be using it in such a generalized types file like this?

Reply:
Renamed it to `SchemaFormConfig`